src/ragas/testset/filters.py

```diff
@@ -57,7 +57,10 @@ async def filter(self, node: Node) -> t.Dict:
         output = results.generations[0][0].text.strip()
         output = await context_scoring_parser.aparse(output, prompt, self.llm)
         output = output.dict() if output is not None else {}
-        output["score"] = sum(output.values()) / len(output.values())
+        try:
+            output["score"] = sum(output.values()) / len(output.values())
+        except ZeroDivisionError:
+            output["score"] = 0  # or some default value or handling
```

Comment:
Relying on exceptions for control flow is usually not efficient. Just use a third operator:

```suggestion
        vals = output.values()
        output["score"] = sum(vals) / n_vals if (n_vals := len(vals)) > 0 else 0
```